docs/en/latest/deployment-guide.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Deployment Guide
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# Overview
+
+This document explains how to support multiple ways to deploy custom plugins.
+
+**Note: This feature is a WIP** 
```

Comment:
What does `this feature` mean? Does it mean deploying custom plugins is still WIP?

Reply:
yes

Reply:
should we remove this docs or change the noto to: `This feature is WIP`?

Reply:
It's up to you

Reply:
fixed